org.uqbar.project.wollok.tests/src/org/uqbar/project/wollok/tests/interpreter/CollectionTestCase.xtend

```diff
@@ -53,6 +55,8 @@ class CollectionTestCase extends AbstractWollokInterpreterTestCase {
 		program p {
 			«instantiateCollectionAsNumbersVariable»		
 			assert.equals(22, numbers.max() )
+			assert.equals(1, [1].max())
+			assert.throwsException({[].max()})
 		}'''.interpretPropagatingErrors
```

Comment:
As a suggestion for the future, it is better to add more tests for different scenarios than adding more assertion in the same test.